sql/core/src/main/scala/org/apache/spark/sql/execution/objects.scala

```diff
@@ -219,17 +218,13 @@ case class MapPartitionsInRWithArrowExec(
     child: SparkPlan) extends UnaryExecNode {
   override def producedAttributes: AttributeSet = AttributeSet(output)
 
-  private val batchSize = conf.arrowMaxRecordsPerBatch
-
   override def outputPartitioning: Partitioning = child.outputPartitioning
 
   override protected def doExecute(): RDD[InternalRow] = {
     child.execute().mapPartitionsInternal { inputIter =>
       val outputTypes = schema.map(_.dataType)
 
-      // DO NOT use iter.grouped(). See BatchIterator.
-      val batchIter =
-        if (batchSize > 0) new BatchIterator(inputIter, batchSize) else Iterator(inputIter)
+      val batchIter = Iterator(inputIter)
```

Comment:
Hmm, I don't see the new iterator is "batched" by `ArrowRRunner`. So it is just removed?

Reply:
Like `ArrowPythonWithNamedArgumentRunner` which is now batching rows internally with `BatchedPythonArrowInput`, but I don't see such thing on ArrowRRunner too. Is it intentional?